frontend/src/app/app-routing.module.ts

```diff
@@ -33,7 +25,23 @@ const routes: Routes = [
         loadChildren: () => import('./modules/session-call/session-call.module').then((m) => m.SessionCallModule),
         canActivate: [AuthGuard],
     },
-    { path: '**', redirectTo: '', pathMatch: 'full' },
+    {
+        path: 'not-found',
+        component: NotFoundComponent,
+        pathMatch: 'full',
+    },
```

Comment:
are you sure we need this path?

Reply:
actually no, I think that I can remove it